pkg/golinters/gofumpt.go

```diff
@@ -33,6 +34,24 @@ func NewGofumpt() *goanalysis.Linter {
 		nil,
 	).WithContextSetter(func(lintCtx *linter.Context) {
 		analyzer.Run = func(pass *analysis.Pass) (interface{}, error) {
+			settings := lintCtx.Settings().Gofumpt
+
+			// Get the target Go language version to be used in gofumpt.
+			// Reuse logic from `https://github.com/mvdan/gofumpt/blob/ba6406b58f3630c5246474b20f6ee5c83d1cc6b9/gofmt.go#L127-L133`.
+			langVersion := settings.LangVersion
+			if langVersion == "" {
+				outGoVersion, errGoVersion := exec.Command("go", "list", "-m", "-f", "{{.GoVersion}}").Output()
```

Comment:
I might be wrong but I was under the impression that `golangci-lint` are not dependent on Go itself. F.ex. when running in CI you can provide a single binary to lint the code. If that's true, this will introduce a new dependency that might impact a lot of users since the default language version is `""`.

Not sure what the solution here would be but maybe worth considering?

Reply:
Thanks for the comment! I think it's reasonable to not introduce a new dependency.
An alternative is to use something similar to `getGoVersion` below used in `StaticCheckSettings`, where we default the version to 1.13 and use the build ReleaseTags later. https://github.com/golangci/golangci-lint/blob/a68a88ee02344ab570cafd5f53ac8d0f5adee299/pkg/golinters/staticcheck_common.go#L17-L29

Reply:
Yeah that sounds like a good alternative! Didn't really read up on the details about this flag but if that works I think the zero dependency part of `golangci-lint` is pretty nice.